gpu_support/nvidia/install_cuda_host_injections.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
Should add some comment here on what this script is meant for. E.g.

```
This script can be used to install CUDA in the host_injections directory. This provides the parts of the CUDA installation that can not be redistributed as part of EESSI due to license limitations. While GPU-based software from EESSI will run without these, installation of additional CUDA software requires the CUDA installations in host_injections to be present.

The host_injections directory is a variant symlink that points to `/opt/eessi`, unless otherwise defined in the local CVMFS configuration (see https://cvmfs.readthedocs.io/en/stable/cpt-repo.html#variant-symlinks). For the installation to be succesful, this directory needs to be writeable to the user executing this script.
```